packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -185,7 +185,7 @@ export default function DrawerNavigator() {
       drawerContent={drawerContent}
       backBehavior={'initialRoute'}
       drawerContentOptions={{
-        labelStyle: [fontStyles.regular, { marginLeft: -20 }],
```

Comment:
without setting weight to normal we get 500 set by react-navigation.

note the margin -20 was here before

Reply:
i also considered setting theis globaly, but the snapshot diff was crazy big so i figured lets be conservative